CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
 
 * Set the `Component.displayName` on all decorators. **NB** You may need to update your snapshots as a result of this to change the component name to something more accurate
 
+# 2.4.1
```

Comment:
should aim for 2.5.0

Reply:
@andrewjtait Ok. I've changed the entry to 2.5.0